code/graphics/color.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
```

Comment:
Generally, we seem to use `#ifndef _FILE_H_` instead of `#pragma once`, would need feedback from others on how they feel about this.